packages/phone-number-privacy/combiner/src/server.ts

```diff
@@ -87,3 +88,47 @@ export function createHandler<R extends OdisRequest>(
 ): PromiseHandler<R> {
   return meteringHandler(catchErrorHandler(enabled ? handler : disabledHandler<R>))
 }
+
+export function startProxy(req: any, res: any, config: CombinerConfig) {
+  let destinationUrl
+
+  const logger = rootLogger(config.serviceName)
+  const proxy = httpProxy.createProxyServer({
+    proxyTimeout: config.phoneNumberPrivacy.odisServices.timeoutMilliSeconds,
+  })
+
+  const originalPath = req.path
```

Comment:
are you just trying to get the endpoint? You should be able to just use req.url for that